src/Utils.php

```diff
@@ -131,6 +131,10 @@ public static function uriTemplate($template, array $variables)
      */
     public static function jsonDecode($json, $assoc = false, $depth = 512, $options = 0)
     {
+        if ($json === '' || $json === null) {
```

Comment:
Both PHP 5 and 7 return null, but in PHP 7 a subsequent json_last_error returns JSON_ERROR_SYNTAX, where 5 returns JSON_ERROR_NONE. See https://3v4l.org/FAO0Z.